frontend/src/components/util/NavBar.vue

```diff
@@ -677,15 +677,16 @@ export default {
       if (this.navBarList?.length > 0) {
         items.push(
           {
-            title: 'Eligibility',
+            title: this.showApplicationTemplateV1 ? 'Eligibility' : 'Facility Information',
             subTitle: this.navBarList[0]?.facilityName,
             subTitle2: this.navBarList[0]?.facilityAccountNumber,
             id: this.navBarList[0].facilityId,
-            link: { name: 'Eligibility GUID', params: { urlGuid: this.navBarList[0].facilityId } },
+            link: { name: 'Family Facility Information GUID', params: { urlGuid: this.navBarList[0].facilityId } },
             isAccessible: true,
             icon: this.getCheckbox(this.navBarList[0].isFacilityComplete),
             isActive:
-              'Eligibility GUID' === this.$route.name && this.$route.params.urlGuid === this.navBarList[0].facilityId,
+              'Family Facility Information GUID' === this.$route.name &&
```

Comment:
I'm not a fan of the verbose route names (as you can see here they look odd) but it seems to be the standard pattern for CCOF.